src/arrayMethodSort.js

```diff
@@ -5,7 +5,35 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = compareAsString) {
```
You can create it and set as default if nothing was passed